crates/types/src/req.rs

```diff
@@ -9,18 +10,8 @@ const DOMAIN_BINDING: &str = "init4.sequencer.v0";
 pub struct SignRequest {
     /// The chain ID of the host.
     pub host_chain_id: U256,
-    /// The chain ID of the rollup.
-    pub ru_chain_id: U256,
-    /// The sequence number of the rollup block.
-    pub sequence: U256,
-    /// The rollup block must be confirmed by this time.
-    pub confirm_by: U256,
-    /// The gas limit of the rollup block.
-    pub gas_limit: U256,
-    /// The reward address for the builder.
-    pub ru_reward_address: Address,
-    /// Encoded transactions to be signed
-    pub contents: B256,
+    /// The Zenith BlockHeader for the rollup block.
+    pub header: ZenithHeader,
```

Comment:
this part in particular i don't know if better or worse

Reply:
for example, previously Quincey resp/req didn't depend on Zenith contract types - not sure if it's positive or negative to introduce that dependency